src/components/Card/Card.jsx

```diff
@@ -1,9 +1,16 @@
-import css from './styles.module.css';
+import { Card } from 'antd';
```

Comment:
можно переименовывать импорты
```suggestion
import { Card as CardAntd } from 'antd';
```
Теперь имя Card свободно и лучше компонент назвать просто Card